example/lstm/README.txt

```diff
@@ -1,19 +1,33 @@
+## GOAL ##
+LSTM example using Platoon.
 
-To use the provided example, follow the steps below :
 
-1) Run the script lstm_master.py in a shell and leave it running until it
-   prints "Controller is ready".  This shouldn't take a long time.
+## CONTENT ##
+- README.txt
+- lstm_master.py
+- lstm_worker.py
+- imdb.py
 
-2) Run "python lstm.py init" until it prints "Param init done".
 
-3) At this point, you can run other workers (with lstm.py).  You may want to
-   run a single worker until it starts training to do all the compilation
-   required and not have all the workers fight for the compile lock.
+## HOW TO USE ##
+# USING THE LAUNCHER
+1) Assuming you are in the lstm folder.
+   `cd platoon/example/lstm/`
 
+1) Launch the experiment on 2 GPUs using the platoon_launcher script.
+   `platoon_launcher lstm gpu0 gpu2`
 
-IMPORTANT
+# MANUALLY
+1) Assuming you are in the lstm folder.
+   `cd platoon/example/lstm/`
 
--You have to manually assign GPUs to the workers (via THEANO_FLAGS=device=???).
+2) Start the controller.
+   `THEANO_FLAGS='device=cpu' python -u lstm_controller.py`
 
--You may want to run them in different windows of screen or tmux since they all
- expect to be in the foreground.
+3) Start the worker. Repeat as needed changing the GPU id.
+   `THEANO_FLAGS='device=gpu0' python -u lstm_worker.py`
```

Comment:
Why are you dropping most of the explanations that were given in the original version?